django_project/ford3/models/provider.py

```diff
@@ -34,12 +34,13 @@ class Provider(models.Model):
         unique=False,
         help_text='',
         max_length=255)
-    logo_url = models.CharField(
-        blank=False,
-        null=True,
-        unique=False,
-        help_text='',
-        max_length=255)
+    # TBC: no longer need this variable
+    # logo_url = models.CharField(
+    #     blank=False,
+    #     null=True,
+    #     unique=False,
+    #     help_text='',
+    #     max_length=255)
```

Comment:
Yes, no need to use this. Delete this from codebase.

Reply:
Please remove the commented code.